manifests/d/Daum/PotPlayer/210729/Daum.PotPlayer.installer.yaml

```diff
@@ -0,0 +1,30 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: Daum.PotPlayer
+PackageVersion: 210729
+FileExtensions:
+  - MKV
+  - OGM
+  - MP4
+  - MOV
+  - 3GP
+  - AVI
+InstallModes:
+  - interactive
+  - silent
+Installers:
+  - Architecture: x64
+    InstallerType: nullsoft
+    InstallerUrl: https://t1.daumcdn.net/potplayer/PotPlayer/Version/210729/PotPlayerSetup64.exe
+    InstallerSha256: FA6E893320CF4A786B2D2EA499231A1E5572B819EA41046D5EE0825C098C3AB5
```

Comment:
```suggestion
    InstallerSha256: e2fa4e970dc5e91c2009ea9771563a8600cfdb8fdd424b5ac44f9b3851e2bd93
```
![image](https://user-images.githubusercontent.com/71691514/127648561-5c67ef9c-b19b-46c0-9353-dedf10e0eccc.png)
Don't know if it will change again or no.